src/ripple/protocol/SystemParameters.h

```diff
@@ -59,6 +60,18 @@ systemCurrencyCode()
 /** The XRP ledger network's earliest allowed sequence */
 static std::uint32_t constexpr XRP_LEDGER_EARLIEST_SEQ{32570};
 
+/** The minimum amount of support an amendment should have.
+
+    @note This value is used by legacy code and will become obsolete
+          once the fixAmendmentMajorityCalc amendment activates.
+*/
+constexpr std::ratio<205, 256> amendmentSuperMajorityThresholdPre3396;
```

Comment:
Shouldn't the numerator be **`204`** to precisely match the semantics of the old code?

Reply:
Thanks. Nasty typo!